package.json

```diff
@@ -70,12 +70,12 @@
     "data-spreadsheet"
   ],
   "dependencies": {
-    "@types/pikaday": "1.6.0",
+    "@types/pikaday": "1.7.4",
```

Comment:
I wonder if `@types` in general shouldn't be in `devDependencies`? 
Because the `@types` is the development tools, not implementations.

Reply:
The types are necessary here. If the developers install `handsontable` into they app the NPM installs only our "dependencies". Hence, when the app is written in TS, without `@types/pikaday` installed developer would get `any` for the `datePickerConfig` options of the date-type cell. Moreover, they would get a compilation error because we use Pikaday types in our definition file.